README.md

```diff
@@ -126,13 +130,19 @@ $ electron . --rpc /my/geth.ipc
 $ electron . --rpc /my/geth.ipc --node-ipcpath /my/geth.ipc
 ```
 
+### Creating a local private net
+
+See this guide to quickly set up a local private network on your computer:
+https://gist.github.com/evertonfraga/9d65a9f3ea399ac138b3e40641accf23
+
+
 ### Using Mist with a privatenet
 
 To run a private network you will need to set the IPC path, network id and data
 folder:
 
 ```bash
-$ electron . --rpc ~/Library/Ethereum/geth.ipc --node-networkid 1234  --node-datadir ~/Library/Ethereum/privatenet
+$ electron . --rpc ~/Library/Ethereum/geth.ipc --node-networkid 1234 --node-datadir ~/Library/Ethereum/privatenet
```

Comment:
We should mention that when you want to start a node outside of Mist with a custom datadir, make sure you set the pic path to the default like:
`geth --datadir custom/... --ipcpath ~/Library/Ethereum/geth.ipc`